src/Sql/dbo/Stored Procedures/ProviderOrganizationProviderDetails_ReadByUserId.sql

```diff
@@ -0,0 +1,23 @@
+﻿CREATE PROCEDURE [dbo].[ProviderOrganizationProviderDetails_ReadByUserId]
+    @UserId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+SELECT
+    PO.Id,
+    PO.OrganizationId,
+    PO.ProviderId,
+    P.Name as ProviderName,
+    P.[Type] as ProviderType
+FROM
+    [dbo].[ProviderOrganizationView] PO
+    INNER JOIN
+        [dbo].[Organization] O ON PO.OrganizationId = O.Id
```

Comment:
You don't need this table join, it's not used anywhere. Make sure to update the EF query as well.